cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -3629,11 +3623,10 @@
 
 ;; Helper fxor creating `pxor` instructions.
 (decl x64_pxor (Xmm XmmMem) Xmm)
-(rule 0 (x64_pxor src1 src2)
-      (xmm_rm_r (SseOpcode.Pxor) src1 src2))
 (rule 1 (x64_pxor src1 src2)
       (if-let true (use_avx))
       (xmm_rmir_vex (AvxOpcode.Vpxor) src1 src2))
+(rule 0 (x64_pxor src1 (is_xmm_mem src2)) (x64_pxor_a src1 src2))
```

Comment:
Is `is_xmm_mem` here necessary, or a copy/paste?

Reply:
Yup, good catch.